README.md

```diff
@@ -510,6 +510,13 @@ The actual lookup edit cell is a fork of the one authored by jlyon87 as found [h
 
 </details>
 
+<details id="soql-datatable---custom-column-labels>
+    <summary>Custom Column Labels</summary>
+
+
+Define custom labels for any column selected. To override, provide a comma-separated list using the `columnLabels` metadata property. Notation uses the Apex Map class's fat arrow notation (`=>`) for mapping the API Name to your new label: `fieldApiName1=>newLabel, fieldApiName2=>new label with spaces 2`. By default, the Label for the Field API Name in question is used.
```

Comment:
Would be good here to also state what is ok and what is not.

e.g. `fieldApiName1=>newLabel` and `fieldApiName1 => newLabel` are both OK (or if the latter is NOT ok, call that out).

Reply:
Cool, will do